### PR TITLE
Add unlocked flag to the dashboard

### LINKED ISSUE #980: Paywall Subscription Status Flag
**Solution**
Once a user has paid for content and the paywall disappears a flag informs them that they've paid for the content.

**Design**
image

Mock: https://www.figma.com/file/LkWSSnCGlKEITBB5T5NYXW/Paywall?node-id=0%3A837

### PR DESCRIPTION
**Pull request by cnasc**
# Description

This very basic PR fixes #980 and puts us in position to tackle #981. It adds the existing `UnlockedFlag` component to the dashboard, so that when the page is in an unlocked state, the flag indicates the subscription status and then hides itself. Once #1338 is handled, the unlocked flag will slide in to be visible on hover, though it doesn't at this moment.

video

_The Demo Page_
screen shot 2019-02-01 at 12 26 21 pm

_The Paywall, locked_
screen shot 2019-02-01 at 12 22 05 pm

_The Paywall, unlocked_
screen shot 2019-02-01 at 12 22 14 pm

_The Paywall, unlocked, with hover_
screen shot 2019-02-01 at 12 22 34 pm

_The Overlay_
screen shot 2019-02-01 at 12 24 40 pm

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [ ] This PR only contains configuration changes (package.json, etc.)
  - [x] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [x] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [x] I have performed a self-review of my own code
- [x] If my code involves visual changes, I am adding applicable screenshots to this thread

PS: [Read how to write the perfect pull request](https://blog.github.com/2015-01-21-how-to-write-the-perfect-pull-request/)

### CODE REVIEW

**Comment by cnasc** [comment]
Actually, now that I've watched that gif several times, it looks like the positioning of the flag is off relative to the flag on the overlay. I'll look into that.

**Comment by cellog** [comment]
having looked at that same question, that positioning is in the original design, so I wouldn't worry about it unless @smombartz thinks you should :)

**Comment by cellog** [comment]
Screenshots look good except for that abomination we haven't fixed yet with eth prices but we will get there on that in a separate PR ;)

File: unlock-app/src/__tests__/pages/paywall.test.js [hunk 1]
```diff
@@ -78,4 +78,28 @@ describe('Paywall', () => {
       expect(unlockPage).toHaveBeenCalled()
     })
   })
+  describe('the unlocked flag', () => {
+    it('should be present when the paywall is unlocked', () => {
+      const { queryByText } = rtl.render(
```
Comment (cellog) [inline unlock-app/src/__tests__/pages/paywall.test.js]: sexy use of destructuring!

Comment (cnasc) [reply unlock-app/src/__tests__/pages/paywall.test.js]: Perhaps my favorite feature in JS right now! (Clojure has good destructuring too)

**Pull request merged**